### PR TITLE
Hi fi: Story Inquiry

### PR DESCRIPTION
**Pull request by jzhzhang66**
*Note*: run npm install 

This PR contains all the styling / form validation for the story inquiry page. 

- StoryConceptFaq.js: component to render the faq section of the page
- StoryInquiryForm.js: component to handle all form rendering / validation 
- StoryInquiryPage.js: component to render entire page including spark arrows, side images, footer, etc.
- Made some changes to quote carousel and confirmation popup so they can be styled on the inquiry page accordingly

### CODE REVIEW

File: components/story-inquiry-page/StoryInquiryPage.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      <H1StyledContainer>
+        <H1Styled
+          regular={storySubmissionTitleRegular}
+          italicized={storySubmissionTitleItalic}
+          storyInquiry={true}
+        />
+        <SquiggleWavy src={icons.SQUIGGLE_WAVY} alt="wavy squiggle underline" />
+      </H1StyledContainer>
+
+      <PageDescription
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/StoryInquiryPage.js:131]: on hi-fi looks like page description is 2 paragraphs, we should use renderRichText in pageDescription, lmk if that should be done in a separate pr

File: components/story-inquiry-page/StoryInquiryPage.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        description={storySubmissionDescription}
+        arrowText={sparkArrowDescriptionText}
+        img={storyInquiryTopImage}
+      />
+      <TopSparkArrowContainer>
+        <LongSparkArrow arrowText={sparkArrowFaqText} />
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/StoryInquiryPage.js]: please add functionality to all long spark arrows, example can be seen [here](https://github.com/Scout-NU/thealleyway-site/pull/123) lmk if you need help

File: components/story-inquiry-page/StoryInquiryPage.styles.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    margin-top: 7.5em;
+  }
+  @media ${min.desktop} {
+    margin-top: 10em;
+  }
+`;
+
+const SideImage = styled.img`
+  object-fit: cover;
+  display: none;
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/StoryInquiryPage.styles.js]: why `display: none`? they should be visible on all screens until mobile

File: components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.js [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+
+  const [trimmedDataUrl, trim] = useState(null);
+  const [sigPad, setSigPad] = useState({});
+  const [isVenmoPolicyOpen, setIsVenmoPolicyOpen] = useState(false);
+  const [isStoryConceptPopupOpen, setIsStoryConceptPopupOpen] = useState(false);
+  const [isConfirmationPopupOpen, setIsConfirmationPopupOpen] = useState(false);
+  const [additionalResources, setAdditionalResources] = useState([{ id: 0, resource: "" }]);
+  const [fields, setFields] = useState({});
+  const [errors, setErrors] = useState({});
+  // console.log(trimmedDataUrl)
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.js]: let's remove this

File: components/story-inquiry-page/StoryInquiryPage.js [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+  SideImageContainer,
+  SideImage,
+  QuoteCarouselContainer,
+} from './StoryInquiryPage.styles';
+import StoryInquiryForm from './story-inquiry-form/StoryInquiryForm';
+import { icons } from '../../style/icons';
+import LongSparkArrow from '../long-spark-arrow/LongSparkArrow';
+import QuoteCarousel from '../quote-carousel/QuoteCarousel';
+
+export default function StoryInquiryPage({
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/StoryInquiryPage.js:27]: If it's an easy fix, let's make the text portions more narrow like the hifi shown below

File: components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.js [hunk 6]
```diff
@@ -0,0 +1,367 @@
...
+          buttonText="SUBMIT MY STORY"
+          long={true}
+          onClick={() => {
+            trim(sigPad.getTrimmedCanvas().toDataURL('image/png'));
+            if (submitRequest()) {
+              document.body.style.overflow = 'hidden';
+              document.getElementById('area').value = '';
+              setFields({});
+              setIsConfirmationPopupOpen(true);
+              console.log(trimmedDataUrl);
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.js]: just a reminder to remove all console.logs before merging

File: components/story-inquiry-page/story-concept-faq/StoryConceptFaq.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
+import { H1, H2, H3, P } from '../../../style/typography';
+import { getString } from '../../../lib/richText';
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/story-concept-faq/StoryConceptFaq.js:2]: not sure where the onclick is to show a faq, but for mobile is it possible for when you click on one & it shows, and if you click the question again it closes?

File: pages/story-inquiry.js [hunk 8]
```diff
@@ -6,93 +6,26 @@ import {
...
-  const storyConceptPopupData = {
-    popupTitles: [popupTitle],
-    popupDescriptions: [popupDescription],
-    popupImage,
-  };
-
-  const boxLinkFooter = { img: footerImage, data: boxLinkDataFooter };
 
   return (
     <>
```
Comment (isabellberzsenyi) [inline pages/story-inquiry.js]: I think we can remove empty tags if just returning `<StoryInquiry ... />`

File: components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.styles.js [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+const InputFieldWrapper = styled.div`
+  padding-bottom: 1.5em;
+`;
+
+const SocialInfoTextWrapper = styled.div`
+  margin-top: -3.8em;
+`;
+
+const SignatureCanvasWrapper = styled.div`
+  background-color: white;
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.styles.js]: let's use `${colors.WHITE}`

File: components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.styles.js [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+  border: 1px solid ${colors.BROWN};
+`;
+
+const TextLabel = styled(P)`
+  color: ${colors.BROWN};
+  display: block;
+  margin: 0;
+`;
+
+const RedStar = styled.img`
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.styles.js]: duplicate code from **TextInputField.style.js** can you use that RedStar? or move it to a shared styled file

File: components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.styles.js [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+const SquareButtonWrapper = styled.div`
+display: flex;
+justify-content: center;
+  @media ${min.tablet} {
+    float: right;
+    margin: -0.3em 0 0 0;
+  }
+`;
+
+const ErrorText = styled(P)`
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.styles.js:117]: same comment as **RedStar** this is duplicate from **TextInputField.style.js**

File: components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.styles.js [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+  position: absolute;
+  color: ${colors.RED};
+  font-size: 0.7em;
+  margin: 0.5em 0 0 0;
+  text-shadow: 0.5px 0 #ff0000;
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.styles.js]: let's use `${colors.RED}` instead of #ff0000

File: components/story-inquiry-page/story-concept-faq/question/Question.js [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+        setHover(true);
+      }}
+        onMouseLeave={() => setHover(false)}
+        hover={hover}
+        onClick={() => {
+          setCurrIndex(index);
+        }}
+        isSelected={isSelected}
+        hover={isSelected ? false : hover}
+      >{getString(question)} {isSelected && <ShortArrow src={icons.FILLED_SHORT_ARROW} alt="filled short arrow" />}
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/story-concept-faq/question/Question.js]: nit: the coding style looks a bit off can you do 
```
{getString(question)}
{
  isSelected && (
  <ShortArrow .... />
)}
```

File: components/story-inquiry-page/story-concept-faq/StoryConceptFaq.styles.js [hunk 14]
```diff
@@ -0,0 +1,47 @@
+
+import styled from 'styled-components';
+import { min } from '../../../lib/responsive';
+import { colors } from '../../../style/colors';
+import {H2, fontWeights} from '../../../style/typography';
+
+const FaqContainer = styled.div`
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/story-concept-faq/StoryConceptFaq.styles.js:4]: nit: can you indent the styling within each object one indent, code style looks off

File: components/story-inquiry-page/story-concept-faq/StoryConceptFaq.js [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+    const [currIndex, setCurrIndex] = useState(questionsAndAnswers[0].id);
+    const currAnswer = questionsAndAnswers.find((item) => item.id == currIndex).answer;
+    return (
+        <FaqContainer>
+            <H2>{getString(faqTitle)}</H2>
+            <QuestionAnswerContainer>
+                <QuestionContainer>
+                    {
+                        questionsAndAnswers.map((item) => {
+                            return <Question key={item.id} index={item.id} questionAnswerData={item} setCurrIndex={setCurrIndex}
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/story-concept-faq/StoryConceptFaq.js]: nit: can you put parens around the Question component and make the component either in one line or multi like below
```
return (
   <Question
      key={item.id}
      index={item.id}
      questionAnswerData={item}
      setCurrIndex={setCurrIndex}
      currIndex={currIndex}
   />
}
```

File: components/story-inquiry-page/StoryInquiryPage.styles.js [hunk 16]
```diff
@@ -0,0 +1,164 @@
...
+
+const LongLinesFlippedDesktop = styled.img`
+display: none;
+@media ${min.desktop} {
+  display: block;
+  position: relative;
+  width: 11.2em;
+  margin: -32em 0 0 4.8em;
+}
+}
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/StoryInquiryPage.styles.js]: indent

File: components/story-inquiry-page/StoryInquiryPage.styles.js [hunk 17]
```diff
@@ -0,0 +1,164 @@
...
+  @media ${min.desktop} {
+    display: none;
+  }
+`;
+
+const LongLinesFlippedTablet = styled.img`
+  @media ${min.tablet} {
+    display: block;
+    position: relative;
+    width: 7em;
```
Comment (isabellberzsenyi) [inline components/story-inquiry-page/StoryInquiryPage.styles.js]: nit & optional: everything in this component is the same as **LongLinesTablet** except for margin, maybe you could do:

```
const LongLinesFlippedTablet = styled(LongLinesTablet)`
    @media ${min.tablet} {
        margin: 0 0 0 0.9em;
   }
`;
```

File: components/quote-carousel/QuoteCarousel.style.js [hunk 18]
```diff
@@ -71,7 +107,6 @@ const QuoteContainer = styled.div`
   left: 2em;
   bottom: 9em;
   margin-bottom: -12em;
-  overflow: hidden;
```
Comment (isabellberzsenyi) [inline components/quote-carousel/QuoteCarousel.style.js:74]: question: just to double check we are "error" checking elsewhere that the quote is not too long?

File: components/home-page/newsletter-sign-up/NewsletterSignUp.styles.js [hunk 19]
```diff
@@ -51,7 +51,11 @@ export const NewsletterSignUpTextInput = styled(TextInputField)`
 `;
 
 export const PrivacyPolicyLinkText = styled.div`
```
Comment (isabellberzsenyi) [inline components/home-page/newsletter-sign-up/NewsletterSignUp.styles.js:53]: is this supposed to be here?

File: components/home-page/newsletter-sign-up/NewsletterSignUp.styles.js [hunk 20]
```diff
@@ -51,7 +51,11 @@ export const NewsletterSignUpTextInput = styled(TextInputField)`
...
+<<<<<<< HEAD
```
Comment (isabellberzsenyi) [inline components/home-page/newsletter-sign-up/NewsletterSignUp.styles.js]: is this supposed to be here?

File: components/h1-styled/H1Styled.styles.js [hunk 21]
```diff
@@ -4,31 +4,64 @@ import { min } from '../../lib/responsive';
...
-    line-height: 0.3em;
   }
   @media ${min.desktop} {
     margin: 0 0 0 0.76em;
     line-height: 0.43em;
   }
 `;
 
-export { H1Container, H1Regular, H1Italicized };
+const H1Italicized1 = styled(H1)`
```
Comment (isabellberzsenyi) [inline components/h1-styled/H1Styled.styles.js:41]: please indent the styling inside of each const like above

File: components/h1-styled/H1Styled.styles.js [hunk 22]
```diff
@@ -4,31 +4,64 @@ import { min } from '../../lib/responsive';
...
   }
 `;
 
 const H1Regular = styled(H1)`
-  margin: 0 auto;
+margin: 0 auto;
+@media ${min.tablet} {
+  margin: ${(props) => (props.longItalicized ? `0 0 0 .2em` : `0 auto`)}; 
+}
+
```
Comment (isabellberzsenyi) [inline components/h1-styled/H1Styled.styles.js]: indent

File: components/story-inquiry-page/StoryInquiryPage.js (hunk 1 above)
Comment (jzhzhang66) [reply components/story-inquiry-page/StoryInquiryPage.js:131]: Yeah I think we can pull that into a separate PR!

File: components/quote-carousel/QuoteCarousel.style.js (hunk 18 above)
Comment (jzhzhang66) [reply components/quote-carousel/QuoteCarousel.style.js:74]: Yes will do a separate PR to make sure the quote carousel's error handling / spacing looks good on both about & story inquiry pages

**Review by isabellberzsenyi — APPROVED** [review]
fabulous

File: components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.js [hunk 23]
```diff
@@ -0,0 +1,452 @@
...
+                  ? fields[fieldNames.FURTHER_EDUCATION_LINK]
+                  : ''
+              }
+              showError={errors[fieldNames.FURTHER_EDUCATION_LINK]}
+              onChange={(e) =>
+                setFields({ ...fields, FURTHER_EDUCATION_LINK: e.target.value })
+              }
+            />
+          </InputFieldWrapper>
+          <InputFieldWrapper>
```
Comment (jean-zhang) [inline components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.js:347]: Missing `cursor: pointer` for the Additional Resources and the Submit Button, and the FAQ links on the left side

File: components/quote-carousel/QuoteCarousel.js [hunk 24]
```diff
@@ -47,20 +52,32 @@ export default function QuoteCarousel({ quoteCarouselData }) {
...
+      <TextWrapper isAuthorTestimonies={isAuthorTestimonies}>
         <TitleWrapper>{getString(title)}</TitleWrapper>
         <DescriptionWrapper>{getString(description)}</DescriptionWrapper>
       </TextWrapper>
-      <CarouselContainer>
+      <CarouselContainer isAuthorTestimonies={isAuthorTestimonies}>
         <ImageArchWrapper>
           <ImageArchContentWrapper>
             <ArchSpark src={icons.FILLED_SPARK_WHITE} />
             <ArchImage url={quoteImage.url} />
```
Comment (jean-zhang) [inline components/quote-carousel/QuoteCarousel.js:63]: Not sure if this PR, but the arch is missing the bottom and the color looks slightly off (we can use transparent instead)
Screen Shot 2021-05-09 at 5 10 33 PM

**Review by jean-zhang — APPROVED** [review]

File: components/quote-carousel/QuoteCarousel.js (hunk 24 above)
Comment (jean-zhang) [reply components/quote-carousel/QuoteCarousel.js:63]: already fixed in separate pr!
